skyportal/utils/offset.py

```diff
@@ -995,15 +895,8 @@ def get_nearby_offset_stars(
     )
 
     # try to get Gaia sources first
-    r = None
-    for retry in range(2):
-        try:
-            g = GaiaQuery()
-            r = g.query(query_string)
-            break
-        except Exception as e:
-            log(f"Gaia query failed: {e}]")
-            time.sleep(1 + 2**retry)
+    with gaia as g:
```

Comment:
No need to wrap this in a try catch or similar?

Reply:
the with itself doesn't need it (nothing in the `__enter__` could cause a failure) but maybe the query could, let me add that.

Reply:
added